application/dash_application/views/pre_exome.py

```diff
@@ -412,10 +452,17 @@ def update_pressed(click,
             passedfilter):
 
         # Apply get selected runs
-        if not runs:
+        if not runs and not instruments and not projects:
             data = pd.DataFrame(columns=empty_bamqc.columns)
         else:
-            data = bamqc[bamqc[BAMQC_COL.Run].isin(runs)]
+            data = bamqc
+
+        if runs:
+            data = data[data[BAMQC_COL.Run].isin(runs)]
+        if instruments:
+            data = data[data[INSTRUMENT_COLS.ModelName].isin(instruments)]
+        if projects:
+            data = data[data[PINERY_COL.StudyTitle].isin(instruments)]
```

Comment:
I think this should be `.isin(projects)`

Reply:
I think you're right! Will update

Reply:
updated